_posts/2018-04-26-beca-software.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: 'Formación pagada: Beca Software (ABIERTA)'
+date: 2018-04-01 11:51:46.000000000 +01:00
+tags: []
+author: guillermo
+show_in_jobs: true
+---
+**Rol**: Becario de programación (Python - SQL)   
+**Industria**: Retail Technology  
+**Localización**: Huesca/Zaragoza (remoto al 80%)  
+**Beca**: 1000€   
+**Duración**: 12 meses  
+**Posibilidades de contratación tras la beca:** Muy altas (hasta ahora 100%)
+
+**SOBRE TI**
+
+Estás cerca de terminar la carrera o la has terminado hace poco y quieres tener una primera experiencia profesional. Antepones aprender a pesar de poder entrar en alguna de las múltiples consultoras o multinacionales que te hacen ofertas. Te gustaría experimentar lo que es trabajar en una start-up. Conoces o has oído hablar sobre las metologías ágiles y la programación extrema y querrías probarlas tú mismo. Sabes algo de Python y/o SQL, o quizá no, pero la tecnología y la programación es tu vocación y encantaría aprender. Te gustaría trabajar en una empresa con un propósito que se amolde a tus valores personales, una empresa ética con una gestión transparente. Te enorgullecería trabajar en una pequeña empresa con un ambicioso proyecto global (citado por el MIT, Harvard Business Review, Standford...) y con compañeros de varias nacionalidades. Quieres estar en un sitio donde se valore tu opinión como la de los demás y que una de las prioridades de tus compañeros sea que estés contento y feliz. Eres responsable y autónomo y, aunque te gusta programar también te gustan muchas otras cosas y por eso valoras tener un horario flexible y trabajar desde casa el 80% de tu tiempo. Estás dispuesto a invertir unos meses en una beca de formación pero al acabar esperas que la empresa te contrate.
+
+**SOBRE NOSOTROS**
+
+El Grupo Frogtek es una empresa social con ánimo de lucro cuyo propósito es iluminar con información el canal tradicional para hacerlo más competitivo (y de paso ayudar a los tenderos de países emergentes y del tercer mundo a competir en mejores condiciones). Lo hacemos, principal aunque no únicamente, ofreciendo una aplicación Android que el tendero puede usar para gestionar y optimizar su negocio, y sostenemos el proyecto a través de la venta estudios de mercado basados en los datos de ventas de los tenderos que almacenamos y procesamos en nuestros servidores en Google y en Amazon.
+
+Nuestra empresa es global y trabaja de forma distribuida. Las operaciones de venta, formación y soporte a tenderos están en México pero la creación de tecnología se ha hecho hasta la fecha en España aunque cada día más trabajamos en remoto y es menos importante dónde puedas estar. Buscamos una persona que venga a reforzar al equipo de tecnología y nos ayude a dar un salto de calidad.
+
+Más información: [https://www.youtube.com/watch?v=BoDtuEUO328](https://www.youtube.com/watch?v=BoDtuEUO328)
+
```

Comment:
If you like to include the video directly can use: `{% include youtube.html youtube_id="BoDtuEUO328" %}`

Reply:
Directly within the markdown??